### PR TITLE
Update only runtime chef dependencies

### PR DESCRIPTION
**Pull request by jkeiser**

### CODE REVIEW

File: bin/appbundle-updater [hunk 1]
```diff
@@ -131,13 +131,15 @@ CHEFDK_APPS = [
   App.new(
     "berkshelf",
     "berkshelf/berkshelf",
-    "guard test",
+    # from chef-dk/version_policy.rb INSTALL_WITHOUT_GROUPS
+    "changelog compat_testing development docgen guard integration maintenance test tools travis style",
```
Comment (lamont-granquist) [inline bin/appbundle-updater]: did you mean to add this to the chef-dk clause below?

Comment (jkeiser) [inline bin/appbundle-updater]: Yep! I'm only testing chef currently, whoops.

**Comment by jkeiser** [comment]
@lamont-granquist fixed!

**Comment by mwrock** [comment]
:+1: if this is producing good ad hoc builds.

**Comment by lamont-granquist** [comment]
👍  looks correct

**Comment by jkeiser** [comment]
Tested in Travis. Merging!

**Pull request merged**